docs/src/main/asciidoc/spring-cloud-square.adoc

```diff
@@ -0,0 +1,163 @@
+= Spring Cloud Square
+include::_attributes.adoc[]
+
+include::intro.adoc[]
+
+NOTE: Spring Cloud is released under the non-restrictive Apache 2.0 license.
+If you would like to contribute to this section of the documentation or if you find an error, you can find the source code and issue trackers for the project at {docslink}[github].
+
+== Quick Start
+
+include::quickstart.adoc[leveloffset=+1]
+
+Spring Cloud Square The project provides Spring Cloud LoadBalancer integration for https://square.github.io/okhttp/[OkHttpClient] and https://square.github.io/retrofit/[Retrofit], as well as a WebClient-backed Retrofit clients.
+
+== OkHttpClient Spring Cloud LoadBalancer integration
+
+An interceptor is added to the `OkHttpClient` created via autoconfiguration which resolves the scheme, host and port from Spring Cloud LoadBalancer and rewrites the url.
+
+This functionality can be accessed by annotating `OkHttpClient.Builder` beans with `@LoadBalanced`:
```

Comment:
Change to:

You can access this functionality by annotating `OkHttpClient.Builder` beans with `@LoadBalanced`: